luatest/server.lua

```diff
@@ -326,6 +326,10 @@ local function wait_for_condition(cond_desc, server, func, ...)
             error(('Timed out to wait for "%s" condition for server (alias: %s, workdir: %s, pid: %d) within %ds')
                 :format(cond_desc, server.alias, fio.basename(server.workdir), server.process.pid, WAIT_TIMEOUT))
         end
+        if not server.process:is_alive() then
```

Comment:
Look at the `Server:stop` function. It seems from time to time we will get the error below when stopping the server.

Reply:
Fixed.
I improved `Server:stop()` after our discussion.